Source/DafnyStandardLibraries/src/DafnyStdLibs/Unicode/Unicode.dfy

```diff
@@ -0,0 +1,17 @@
+/*******************************************************************************
+ *  Copyright by the contributors to the Dafny Project
+ *  SPDX-License-Identifier: MIT
+ *******************************************************************************/
+
+/**
+ This module defines concepts from the core specification
+ of the [Unicode Standard 15.1.0](https://www.unicode.org/versions/Unicode15.1.0/):
+ scalar values, code points, important kinds of code point sequences,
+ Unicode encoding forms and encoding schemes (including UTF-8 and UTF-16),
+ and helpful properties and functions for manipulation and conversion of Unicode strings.
+ */
+module DafnyStdLibs.Unicode {
+  // This top-level module contains no definitions.
+  // It exists to enable import between child modules,
+  // and so that its doc comment appears in generated documentation.
```

Comment:
This is a good point and too bad we can't do the same for the top level `DafnyStdLibs` module since it has to appear in multiple doo files. But could hopefully address that somehow in the future.